deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -976,6 +976,7 @@ class ChatViewController: UITableViewController {
         let chat = dcContext.getChat(chatId: self.chatId)
         if chat.isSelfTalk {
             RelayHelper.sharedInstance.forward(to: self.chatId)
+            refreshMessages()
```

Comment:
wondering why that is not needed below.

Reply:
I bet there's a event missing if dc_forward_msgs is called for the self chat